src/pgduckdb_options.cpp

```diff
@@ -65,6 +65,9 @@ ReadDuckdbSecrets() {
 		if (!is_null_array[Anum_duckdb_secret_region - 1])
 			secret.region = DatumToString(datum_array[Anum_duckdb_secret_region - 1]);
 
+		if (!is_null_array[Anum_duckdb_secret_session_token - 1])
+			secret.region = DatumToString(datum_array[Anum_duckdb_secret_session_token - 1]);
```

Comment:
I'm not fully familiar, but interested in this feature.  

This is setting the session token into region.  Likely a copy/paste from the lines above?